common/test/acceptance/pages/lms/teams.py

```diff
@@ -230,9 +235,9 @@ def sort_teams_by(self, sort_order):
         self.wait_for_ajax()
 
 
-class CreateOrEditTeamPage(CoursePage, FieldsMixin):
+class CreateEditDeleteTeamPage(CoursePage, FieldsMixin):
```

Comment:
Nit: This is quite a mouthful. Could it be called the TeamEditorPage or TeamManagementPage?